CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Ensure `color-mix(…)` polyfills do not cause used CSS variables to be removed ([#17555](https://github.com/tailwindlabs/tailwindcss/pull/17555))
 - Ensure the `color-mix(…)` polyfill creates fallbacks for theme variables that reference other theme variables ([#17562](https://github.com/tailwindlabs/tailwindcss/pull/17562))
 - Fix brace expansion in `@source inline('z-{10..0}')` with range going down ([#17591](https://github.com/tailwindlabs/tailwindcss/pull/17591))
+- Fix `fontSize` plain array tuple upgrade ([#17630](https://github.com/tailwindlabs/tailwindcss/pull/17630))
```

Comment:
```suggestion
- Ensure compatibility with array tuples used in `fontSize` JS theme keys ([#17630](https://github.com/tailwindlabs/tailwindcss/pull/17630))
```
Not 100% sure about this change but I want to make sure this has to do with JS theme keys so it's clearer who this fix is for. WDYT?

Maybe this is a bit more clear

Reply:
Sure! Perhaps even having two changelog entries? As some may not understand that this also fixes this in the upgrade tool. So have one changelog entry mentioning better compatibility with in JS compate layer and then another entry saying that the `fontSize` array tuple is fixed for the upgrade tool.

Reply:
two sound good!